erpnext/controllers/taxes_and_totals.py

```diff
@@ -58,12 +58,23 @@ def _calculate(self):
 		self.initialize_taxes()
 		self.determine_exclusive_rate()
 		self.calculate_net_total()
+		self.calculate_tax_withholding_net_total()
```

Comment:
Since you have entered this field just for purchase invoices, this method can be called inside validate event in `purchase_invoice.py`